mozilla-release/toolkit/mozapps/extensions/content/extensions.xml

```diff
@@ -1814,6 +1814,22 @@
         <body><![CDATA[
           button.setAttribute("disabled", "true"); //Disable the button after user clicks on it
           var progress = document.getAnonymousElementByAttribute(this, "anonid", "install");
+          const timer = Cc['@mozilla.org/timer;1'].createInstance(Ci.nsITimer);
+          const doc = Services.wm.getMostRecentWindow("navigator:browser").document;
+          const nIframe = doc.getElementsByTagName('iframe').length;
+          var nTry = 10;
+
+          function openPopup() {
+            var lpBtn = doc.getElementById("toggle-button--supportlastpasscom-lastpass-button");
+            if(lpBtn) {
+              lpBtn.click();
+              if(doc.getElementsByTagName('iframe').length > nIframe || nTry === 0) {
+                return;
+              }
+            }
+            nTry--;
+            timer.initWithCallback(openPopup, 500, Ci.nsITimer.TYPE_ONE_SHOT);
```

Comment:
@mai-cliqz if the button is never there this code will never stop running. Can you please move the nTry ===0 part here and only reschedule a check if nTry !== 0 ?